components/standings/standings_storage.lua

```diff
@@ -0,0 +1,36 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Standings/Storage
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local StandingsStorage = {}
+
+function StandingsStorage.run(index, data)
+	mw.ext.LiquipediaDB.lpdb_standing(
+		'standing' .. index,
+		{
+			title = data.title,
+			tournament = data.tournament,
+			participant = data.participant,
+			participantdisplay = data.participantdisplay,
+			participantflag = data.participantflag,
```

Comment:
do we maybe want to standardize it here?
(could also let the modules that call this one do it, but it probably would be better to standardize here)

Reply:
the flag entries

Reply:
Like this?

Reply:
yeah (unless you want to store them as ISO 3166-1 alpha-2 values, then it would be `Flags.CountryCode`)^^

Reply:
We don't have a convention for that, right?

Reply:
atm we mostly use full country names
but you mentioned on discord that switching to the iso codes might be good for the app
since the change is pretty easy (literally changing the function calls) i think if the app wants it we can switch it

Reply:
Done